Sources/RunnerLib/VersionChecker.swift

```diff
@@ -0,0 +1,50 @@
+import DangerShellExecutor
+import Logger
+import Version
+import Foundation
+
+public enum VersionChecker {
+    public static func checkForUpdate(current currentVersionString: String) {
+        if ProcessInfo.processInfo.environment["DEBUG"] != nil {
```

Comment:
```suggestion
        guard ProcessInfo.processInfo.environment["DEBUG"] == nil else {
```

Nitpick

Reply:
I would like if possible another env variable which if set can opt out this check, so folks can disable if they want